### PR TITLE
redirect anonymous user to courses page on root

### PR DESCRIPTION
**Pull request by sheralim012**
#### Story Link
https://edlyio.atlassian.net/browse/EDE-1427

#### PR Description
redirect anonymous user to courses page

#### Type of change

Please select the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Change

#### How to test?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configurations.

- [ ] go to `/` as anonymous user
- [ ] user will be redirected to `/courses`

#### Checklist before merging:

- [ ] Squased
- [ ] Reviewd

### CODE REVIEW

**Comment by sheralim012** [comment]
jenkins run python

File: lms/djangoapps/branding/tests/test_views.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # encoding: utf-8
 """Tests of Branding API views. """
+import copy
```
Comment (emzubair) [inline lms/djangoapps/branding/tests/test_views.py:3]: since we are only using `deepcopy` change the import as 
`from copy import deepcopy`

**Review by emzubair — CHANGES_REQUESTED** [review]

File: lms/djangoapps/branding/tests/test_views.py (hunk 1 above)
Comment (sheralim012) [reply lms/djangoapps/branding/tests/test_views.py]: done

**Review by emzubair — APPROVED** [review]
LGTM 👍 
BTW test job is failing, make sure it's not because of your changes

**Comment by sheralim012** [comment]
> LGTM 
> BTW test job is failing, make sure it's not because of your changes

it's not failing because of my changes also I created an other PR for its fix https://github.com/ucsd-ets/edx-platform/pull/448

**Comment by sheralim012** [comment]
jenkins run python

**Comment by sheralim012** [comment]
jenkins run python

**Comment by sheralim012** [comment]
jenkins run python

**Review by mumerfarooq000 — APPROVED** [review]

**Pull request merged**